### PR TITLE
Update zwave_js.markdown

### PR DESCRIPTION
**Pull request by gleversluis**
Added prereq HA OS or Supervisor for clarity.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zwave_js.markdown
Comment (frenck) [inline source/_integrations/zwave_js.markdown:60]: - The installation method is called "Home Assistant Operating System", not Home Assistant OS.
- The Supervisor will run on OS and cannot be run separately, I think you meant the "Home Assistant Supervised" installation method, and not the Supervised application.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Marked it draft/requested changes again. You need to make the changes as suggest above before you mark it ready for review.

Thanks 👍 

../Frenck

**Comment by gleversluis** [comment]
You know what? I'll close it. Reject the PR for all I care... and don't expect more contributions from me unless you're ready to appreciate that someone is trying to contribute, instead of nitpicking on everything.

**Comment by frenck** [comment]
Wait what? 😕 

Are you OK? Review comments are a very normal and natural thing in a PR process. It keeps more eyes on things and ensures we keep everything correct and quality high. This isn't personal.

If I make a PR myself, I will go through the same process, and I hope someone actually does improve my writing or code each time I do make a PR. I can only learn from it, and makes the end-result better.

It is a small change that is suggested in my review comment above to make it correct. If you think it is non-sense to ship correct changes, please... by all means, don't contribute 🤷 

> unless you're ready to appreciate that someone is trying to contribute

We appreciate your contributions very much! This has nothing to do with that in any way. I'm not sure how you ended up with that conclusion.

But yeah, if this is how you think open source works, I would strongly suggest not making PRs in the future. If you want to change your mind, feel free to re-open, I'm happy to continue on this PR. If you need help or guidance on making that change, we are happy to help too.

../Frenck

File: source/_integrations/zwave_js.markdown
Comment (gleversluis) [reply source/_integrations/zwave_js.markdown:60]: @frenck I thought about messaging you directly, but because I find it important to make a more generic point, I'm replying here. I know how open source works. I know about pull requests, branches, commits and so on - they're part of my day-to-day work, although not a big part since I'm not a full-time software developer.

In this case, all I tried was to make the doc just that more readable, based on this issue: https://github.com/home-assistant/home-assistant.io/issues/28062. In a comment on this, @MartinHjelmare stated that the doc "assumes you're running Home Assistant OS or Supervisor which can install add-ons". I used those exact terms. And I'm not the only one on this exact doc page: under the Advanced installation instructions, the term Home Assistant OS is also used.

It is a small clarification. Not a code change, just a remark in the documentation to move away from assumptions - just because many people know what assumptions are: the mother of all... foul-ups. In this case, an assumption that cost me a few hours of trying to get something to work, only to find out it never will.

The attitude towards contributions in general must be one of seeing the value. Not one of nitpicking on semantics in a minor doc contribution. Your review is the complete opposite of that - something I see in many, many contributions both here on Github and in the various forums. I have seen quite a few people mentioning that because of the reactions they get, they now run other platforms. And that is my generic point: as far as I'm concerned, your comments are an example of how not to treat a contribution.

Comment (frenck) [reply source/_integrations/zwave_js.markdown:60]: > Your review is the complete opposite of that 

Sorry, but you are entirely pulling this out of context, and I don't understand why you see a review as something negative. See my comment below here for earlier, which fully applies as a response to your message here:

<https://github.com/home-assistant/home-assistant.io/pull/28063#issuecomment-1619966934>

I'm was here to help, not to punch you in the face.

> And that is my generic point: as far as I'm concerned, your comments are an example of how not to treat a contribution.

That comment makes no sense whatsoever, are you saying we should just accept and merge everything, regardless if it is correct or not? 🤷 

Let's be real now... I suggest two tiny tweaks to this line to make the wording correct. You are blowing it now out of proportion to make some point that I don't get. You could have just made a small change instead. I would have approved and seriously thanked you for your contribution; and above all, the docs would have a correct and good addition for others in the future.

Instead, you've chosen that making a small change to make it correct is offensive, and closing the PR is a better way to make your (non-existing) point. 🤷 

This is not on me.

../Frenck

Comment (gleversluis) [reply source/_integrations/zwave_js.markdown:60]: I don't see _a_ review as something negative per se. I see _your_ review as something negative.

> That comment makes no sense whatsoever, are you saying we should just accept and merge everything, regardless if it is correct or not? 🤷

Absolutely not. But you're stating "Home Assistant OS" is incorrect, and so is "Supervisor". That wording may not be perfect but it is by no means incorrect. However, you apparently think it's necessary to use your power to not approve the PR, and to treat me as a little schoolboy by telling me to go back and make the wording exactly as _you_ want it. The message that you're sending by doing so, is that my work isn't good enough in the eyes of the Lord Frenck Almighty. _That_ is what started all of this and it _is_ on you.

Comment (ScottG489) [reply source/_integrations/zwave_js.markdown:60]: I think he was just making a correction in line with our documentation standards: https://developers.home-assistant.io/docs/documenting/standards#general-documentation

> The case of brand names, services, protocols, integrations and platforms must match its respective counterpart. E.g., "Z-Wave" not "Zwave", "Z-wave", "Z Wave" or "ZWave". Also, "Input Select" not "input select" or "Input select".

I'd also suggest you read this section in response to getting feedback on PRs: https://developers.home-assistant.io/docs/review-process#receiving-review-comments

> Be very aware these review comments are not personal. The reviewer is not trying to insult you or make you feel bad. They are trying to help you improve your PR, so it can be merged.

While PR comments/corrections can come across as negative, he was just being direct. Frenck reviews a lot of PRs and I can see it becoming exhausting if he tried to sugar coat everything.

Comment (frenck) [reply source/_integrations/zwave_js.markdown:60]: > but it is by no means incorrect. 

It is. Sorry, hence my comment. Those refer to application parts, not installation methods. If you think we should have reference application parts, the suggestion in this PR would have been a duplicate, as the OS cannot function without the Supervisor application; which would have made the mention of the Supervisor application obsolete.

> However, you apparently think it's necessary to use your power to not approve the PR

As a reviewer, like any other reviewer, it is my task to verify the PR, and yes, just like any other reviewer, I will ask for changes in case we've seen an issue with it. In this case, I've provide a small suggestion in the wording used.

>  and to treat me as a little schoolboy by telling me to go back and make the wording exactly as you want it.

I did no such thing; I left a review comment pointing out something was not fully correct in your suggestion here: <https://github.com/home-assistant/home-assistant.io/pull/28063#discussion_r1251803317>, and even provided the suggested wording to use instead.

The bot explained what is expected next and links to a document with detailed information on the review process; what is means and entails. What happened next, is that you ignored that, and just marked the PR ready for review (without any change), dismissing my review comment apparently (without adding any reasoning or context). Therefore, I marked it back to draft, as there is something to change, discuss, or help out with. The latter options didn't happen, as you decided to close the PR instead.

> is that my work isn't good enough in the eyes of the Lord Frenck Almighty

That is correct, it wasn't correct. You seem not be able to see that?

Anyways, that comment is close to getting personal and mocking behavior. Please don't go that route, we have a code of conduct that disallow such behavior in our community, which also applies to our GitHub repositories.

>  That is what started all of this and it is on you.

Sorry, it isn't. You don't have to agree; that is fine. But there was nothing out of the ordinary or wrong with this code review. Your response, however, has been slightly out of the ordinary.

Your behavior is noticed in the community as well, as it was discussed in our Discord chats by others. From that perspective, it was useful, as another person stood up and actually did make the suggested correction, which has been merged and published.

Although I really rather have merged the original PR after a tweak was made.

../Frenck

Comment (Petro31) [reply source/_integrations/zwave_js.markdown:60]: @gleversluis it took me less than 5 minutes to change "Home Assistant OS" to "Home Assistant Operating System" and create a new PR, submit it, and merge it.  You've spent more time chastising others than it would have taken to fix the PR.  Your response was so outlandish, that it took notice to many others in the community.  Please leave your ego at the door in the future when creating PRs.  Reviews and comments are quite normal during the PR creation process.  Getting overly upset about a review or comment is not normal.

**Pull request closed without merging**
